internal/ci/go_test

```diff
@@ -3,11 +3,9 @@
 set -e
 
 go get -u github.com/smartcontractkit/goverage
-
-cc-test-reporter before-build
+sleep 1
```

Comment:
Why is the sleep required?

Reply:
I ran into a test failure where `goverage` failed to run because `text file was busy`: https://stackoverflow.com/questions/16764946/what-generates-the-text-file-busy-message-in-unix

Thought `go get` wasn't quite done with it and placed that sleep. I can remove it and try again if you would like.